README.md

```diff
@@ -18,7 +18,6 @@ Supports the following platforms:
  * iOS
  * Windows
 
-It also compiles for Android but never been tested on it.
 
```

Comment:
You can even remove one extra empty line. ;)

Reply:
Thanks. Addressed with removal.